lib/jekyll-import/importers/drupal6.rb

```diff
@@ -41,10 +43,12 @@ def self.post_data(sql_post_data)
         content = sql_post_data[:body].to_s
         summary = sql_post_data[:teaser].to_s
         tags = (sql_post_data[:tags] || "").downcase.strip
+        permalink = "/" + sql_post_data[:alias]
```

Comment:
This should new conditional, right? In case the data `sql_post_data[:alias]` isn't there?